### PR TITLE
Handle query array params

### PR DESCRIPTION
**Pull request by codetheweb**
Tested with the example app and both `?ids=id-1,id-2` and `?ids=id-1&ids=id-2` work.

### CODE REVIEW

**Review by seveibar — APPROVED** [review]
WOAH nice!!!!

File: apps/example-todo-app/pages/api/todo/list.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        id: z.string().uuid(),
+      })
+      .array(),
+  }),
+})
+
+export default withRouteSpec(route_spec)(async (req, res) => {
+  const { ids } = req.commonParams
+
+  console.log({ ids })
```
Comment (seveibar) [inline apps/example-todo-app/pages/api/todo/list.ts:25]:
```suggestion
```

Comment (seveibar) [reply apps/example-todo-app/pages/api/todo/list.ts:25]: ehh maybe leave it since there are no tests here

Comment (codetheweb) [reply apps/example-todo-app/pages/api/todo/list.ts]: added a test

**Review by seveibar — APPROVED** [review]

**Pull request merged**